### DISCUSSION TITLE
Lighthouse 10 - Pagespeed, problems with CMP

### DISCUSSION THREAD

**Discussion by onlygio** [GoogleChrome/lighthouse#14898] [General]
Hello, 
all my websites had a score > 90 in pagespeed insight. After the updates it's drastically lower. What is happened is that now the CMP modal was included in the results, so every params is wrong.

I usually show the cmp after the page was completely rendered and it worked since 2 days ago. There is someone with my same situation? Is there a workaround? Someone can tell me a path to follow for managing this situation?

NOW
<img width="773" alt="Immagine" src="https://user-images.githubusercontent.com/870665/225601438-44a6b969-450d-4921-8166-48a31db8f5b6.png">

The same 3 days ago:
![WhatsApp Image 2023-03-07 at 17 27 07](https://user-images.githubusercontent.com/870665/225601229-b04acac0-3225-4cf7-bfe7-dbaa247abaec.jpeg)

**Comment by adamraine** [comment]
Can you provide a URL for us to reproduce this with?

**Reply by onlygio** [reply]
yes of course, you can try with: https://www.mondofuoristrada.it/2023/03/16/carlos-sainz-paura-padre-figlio-terrore-occhi-video/

Thanks for your time

**Reply by adamraine** [reply]
Lighthouse is working correctly here. Your page is serving different content for Lighthouse v10.0.2 than it was for v9.6.8.

If I had to wager a guess, your page is choosing not to show the CMP if it detects the [Lighthouse user agent](https://github.com/GoogleChrome/lighthouse/blob/91d457b0abc44d5adea547b815ec02498d8859df/core/config/constants.js#L83) loading the page. This user agent was updated in v10.

In general we discourage serving different content for Lighthouse than what an actual user would see. If you want to test the page as a returning user would see it (i.e. after accepting the CMP), I recommend designing a [user flow](https://github.com/GoogleChrome/lighthouse/blob/main/docs/user-flows.md).
